src/screens/MyAccount.tsx

```diff
@@ -18,9 +19,9 @@ const styles: { [identifier: string]: CSSProperties } = {
 
 export default function MyAccount(): ReactElement {
   const authUser = useAuthUser();
-  // TODO: https://promopal.atlassian.net/browse/PP-80
+
   if (!authUser) {
-    return <p>Error: No user is logged in.</p>;
+    return <Redirect to="/account" />;
```

Comment:
Ah this is a much better idea! This would essentially refresh the page right?

Reply:
I think so yeah -- technically this is one of those weird cases because we shouldn't even be on this page if authUser is not defined (since you guarded against it in Router.tsx)